src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java

```diff
@@ -436,6 +479,7 @@ public void replaceFiltersWithScriptTest() throws Exception {
                 .contentType(APPLICATION_JSON))
                 .andExpect(status().isOk())
                 .andExpect(content().contentTypeCompatibleWith(APPLICATION_JSON));
+        cleanDB();
```

Comment:
useless because it's already in the setup

Reply:
Done